backendservice/src/test/java/com/google/gpay/virtualqueue/backendservice/repository/InMemoryVirtualQueueRepositoryTest.java

```diff
@@ -70,4 +77,19 @@ public void testGetWaitingTime() throws Exception {
         
         assertEquals("Waiting Time per customer is", WAITING_TIME_MINS, waitingTime);
     }
+
+    @Test
+    public void testGetTokens_success() throws Exception {
```

Comment:
Get tokens for a shop right? Make that evident in the name

Reply:
done!